### DISCUSSION TITLE
(Feature Request) (Video) Ability to lock individaul nodes

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1568] [Ideas]
https://github.com/freeplane/freeplane/assets/147225942/3a45ba5f-de1e-4496-b28a-5cacf041e93e

**Comment by EdoFro** [comment]
In the mean time:

Tab "Behaviour" in the preferences dialog has many options that radically change the user experience.

For example I use `Preferences…->Behaviour->On key type` `Add sibling node` to avoid unintentional node's text changes.

It may help.

**Reply by adamfreeplaneuser01** [reply]
Oh wow, that's a great tip! Without this comment, there's no way I would've known about that. I'd still like to see a "lock node" feature, but this is a great alternative in the meantime.

**Comment by adamfreeplaneuser01** [comment]
@dpolivaev Hi! Tagged you to follow up on this feature request. I hope it can be added in. Thanks!

**Reply by dpolivaev** [reply]
Why is the solution suggested by Edo not good enough?

**Reply by adamfreeplaneuser01** [reply]
It was a great suggestion. But "locking" a node offers an extra level of "peace of mind".
